lib/map_elevation.dart

```diff
@@ -269,12 +267,10 @@ class ElevationGradientColors {
 }
 
 /// Geographic point with elevation
-class ElevationPoint {
-  /// Latitude and Longitude
-  lg.LatLng latLng;
```

Comment:
Please add a getter for backward compatibility to retrieve an instance of `LatLng` (basically `this`)